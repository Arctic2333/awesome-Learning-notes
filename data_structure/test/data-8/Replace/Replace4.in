在夜晚的天空中总是有一些闪闪发光的东东飘在天上，其中大部分是星星，但也可能是飞机，可能是孔明灯，还可能是造访地球UFO，现在你的任务是要输出其中有多少个UFO。
可能是
有的是