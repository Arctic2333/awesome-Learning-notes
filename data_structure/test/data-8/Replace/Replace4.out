在夜晚的天空中总是有一些闪闪发光的东东飘在天上，其中大部分是星星，但也有的是飞机，有的是孔明灯，还有的是造访地球UFO，现在你的任务是要输出其中有多少个UFO。
